ray-operator/controllers/ray/common/constant.go

```diff
@@ -125,6 +125,26 @@ const (
 
 	// Finalizers for RayJob
 	RayJobStopJobFinalizer = "ray.io/rayjob-finalizer"
+
+	// Belows are Options names for Ray start
```

Comment:
I prefer not to modify the `rayStartParams` options in this PR. These options appear in hundreds of places throughout the KubeRay codebase, necessitating a significant amount of time to verify the correctness of any changes. Moreover, I have not observed any instances of a `rayStartParams` option being deprecated in the past year. We can discuss the changes to `rayStartParams` further in the future, but it is good to keep this PR small and focused on port names.

Reply:
Really thanks for your valuable suggestions.